e2e/pages/strategy.spec.ts

```diff
@@ -13,71 +14,16 @@ test.describe('Strategies', () => {
-    await navigateTo(page, '/');
-    await page.getByTestId('create-strategy-desktop').click();
-
-    // Select Base
-    await page.getByTestId('select-base-token').click();
-    await waitFor(page, 'modal-container');
-    await page.getByLabel('Select Token').fill('eth');
-    await page.getByTestId('select-token-ETH').click();
-    await page.getByTestId('modal-container').waitFor({ state: 'detached' });
-
-    // Select Quote
-    await page.getByTestId('select-quote-token').click();
-    await waitFor(page, 'modal-container');
-    await page.getByLabel('Select Token').fill('dai');
-    await page.getByTestId('select-token-DAI').click();
-    await page.getByText('Next Step').click();
-
-    // Fill Buy fields
-    const buy = page.getByTestId('buy-section');
-    await buy.getByTestId('input-limit').fill('1500');
-    await buy.getByTestId('input-budget').fill('10');
-    await expect(buy.getByTestId('outcome-value')).toHaveText('0.006666 ETH');
-    await expect(buy.getByTestId('outcome-quote')).toHaveText('1,500 DAI');
-
-    // Fill Sell fields
-    const sell = page.getByTestId('sell-section');
-    await sell.getByTestId('input-limit').fill('1700');
-    await sell.getByTestId('input-budget').fill('2');
-    await expect(sell.getByTestId('outcome-value')).toHaveText('3,400 DAI');
-    await expect(sell.getByTestId('outcome-quote')).toHaveText('1,700 DAI');
-
-    await page.getByText('Create Strategy').click();
-
-    // Accept approval
-    const approvalModal = await waitFor(page, 'approval-modal');
-    const ethMsg = approvalModal.getByTestId('msg-ETH');
-    await expect(ethMsg).toHaveText('Pre-Approved');
-    await approvalModal.getByTestId('approve-DAI').click();
-    const daiApprovalMsg = await waitFor(page, 'msg-DAI');
-    await expect(daiApprovalMsg).toHaveText('Approved');
-    await approvalModal.getByText('Create Strategy').click();
-
-    await page.waitForURL('/', { timeout: 10_000 });
-
-    // Verfiy notification
-    const notif = page.getByTestId('notification-create-strategy');
-    await expect(notif.getByTestId('notif-title')).toHaveText('Success');
-    await expect(notif.getByTestId('notif-description')).toHaveText(
-      'New strategy was successfully created.'
-    );
-
-    // Verify strategy data
-    const strategies = page.locator('[data-testid="strategy-list"] > li');
-    await strategies.waitFor({ state: 'visible' });
-    await expect(strategies).toHaveCount(1);
-    const [strategy] = await strategies.all();
-    await expect(strategy.getByTestId('token-pair')).toHaveText('ETH/DAI');
-    await expect(strategy.getByTestId('status')).toHaveText('Active');
-    await expect(strategy.getByTestId('total-budget')).toHaveText('$3,344');
-    await expect(strategy.getByTestId('buy-budget')).toHaveText('10 DAI');
-    await expect(strategy.getByTestId('buy-budget-fiat')).toHaveText('$10.00');
-    await expect(strategy.getByTestId('sell-budget-fiat')).toHaveText('$3,334');
+  testCreateLimitStrategy({
```

Comment:
I actually just meant:

```ts
const config =  {
  base: 'ETH',
  quote: 'DAI',
  buy: {
    price: '1500',
    budget: '10',
  },
  sell: {
    price: '1700',
    budget: '2',
  },
};
test(`Create Limit Strategy ${config.base}->${config.quote}`, async ({ page }) => {
...
```

I wouldn't put the test case into util - sorry if my earlier comment was confusing

Reply:
This works only if there is one test. Once I add more test into the describe I'll have to prefix the config or add a new scope. I would rather keep the config in the scope and hard code the name, as it was before, if it's ok with you.

Reply:
Sure, but please take the test case out of utils